app/indexers/facet_indexer.rb

```diff
@@ -48,6 +49,14 @@ def structure?
     false
   end
 
+  def pub_date_start
+    date = resource.primary_imported_metadata.created
+    return unless date
+    date = date.first
+    date = Time.zone.parse(date) unless date.respond_to?(:year)
```

Comment:
Should we capture parsing errors here or do you think we should just wait for it to happen? Could end up failing to save objects.